docs/developer/modules/04-cyanregistry.md

```
@@ -0,0 +1,127 @@
+- `cyanregistry/src/cli/` - CLI-specific models
+
+## Responsibilities
+
+- Fetch templates from registry
+- Push templates, plugins, processors
+- Query template metadata
+- Map between domain and HTTP models
+
+## Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)